tests/e2e/test_e2e.py

```diff
@@ -196,10 +198,6 @@ def test_e2e():
     temp_ngrams = MentionNgramsTemp(n_max=2)
     volt_ngrams = MentionNgramsVolt(n_max=1)
 
-    Part = mention_subclass("Part")
```

Comment:
This is awesome! We can also eliminate this part in incremental test as well.

Reply:
Done by b9497adb606709537ed617c80cd37a89942fe6df.